oraclepaas/resource_mysql_service_instance_test.go

```diff
@@ -0,0 +1,235 @@
+package oraclepaas
+
+import (
+	"fmt"
+	"github.com/hashicorp/go-oracle-terraform/mysql"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"os"
+	"testing"
+)
+
+/** Use these variables for OCI Testing. Fill in the appropriate values */
+var (
```

Comment:
We should read these from the environment instead of setting them here

Reply:
Ok. I've added a bunch of enivironment parameters for setting these parameters. 
This will need to be documented ?